src/components/create-guild/hooks/useSetImageAndNameFromPlatformData.ts

```diff
@@ -23,7 +23,13 @@ const useSetImageAndNameFromPlatformData = (
 
   useEffect(() => {
     if (!onUpload) return
-    if (!(platformImage?.length > 0) || !!touchedFields.imageUrl) {
+
+    if (platformImage && platformImage.includes("/guildLogos")) {
```

Comment:
I think we can remove this extra logic once you implement the changes which I mentioned in my comment inside `DiscordGuildSetup.tsx`